sentry_streams/sentry_streams/examples/events.py

```diff
@@ -1,25 +1,50 @@
 import json
 from dataclasses import dataclass
-from typing import Any, Self
+from typing import Any, Generator, Self, Union
 
-from sentry_streams.pipeline.function_template import Accumulator
+from sentry_streams.pipeline.function_template import Accumulator, GroupBy
 
 
 @dataclass
 class Event:
     project_id: int
     latency: int
     timestamp: int
+    tags: list[str]
+    value: int
 
 
 @dataclass
-class AlertData:
+class AlertingEvent:
+    alert_id: int
+    latency: int
+    alert_type: str
+
+
+@dataclass
+class p95AlertData:
```

Comment:
When I mentioned the p95 I meant it as an example. Did not mean to push you to build it. 
If you feel doing count only would make things simpler please feel free to remove p95. Up to you.